yogstation.dme

```diff
@@ -3014,6 +3014,7 @@
 #include "yogstation\code\game\objects\items\cards_ids.dm"
 #include "yogstation\code\game\objects\items\clown_items.dm"
 #include "yogstation\code\game\objects\items\crayons.dm"
+#include "yogstation\code\game\objects\items\cyborg_shit.dm"
```

Comment:
Really going to need you to modify this filename for consistency. 
cyborg_shit.dm  doesn't provide any information as to what it is. 
There is a reason we don't name our files peepeepoopoo_cyborg_fuck.dm

The codebase isn't for your shitposting.

Reply:
renamed to bettertools